tests/features/common/kie-kieserver-common.feature

```diff
@@ -483,3 +483,12 @@ Feature: Kie Server common features
       | OPTAPLANNER_SERVER_EXT_THREAD_POOL_QUEUE_SIZE | 4     |
     Then container log should contain -Dorg.optaplanner.server.ext.thread.pool.queue.size=4
 
+  Scenario: RHPAM-3211 Openshift properties related to passwords in EJB_TIMER cannot use literal $n
```

Comment:
EJB timer only applyes to rhpam, move to rhpam-kieserver.feature.

Reply:
ok